src/shims/shims.proj

```diff
@@ -1,17 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <Import Project="Directory.Build.props" />
-
   <PropertyGroup>
     <PackageConfigurations>
-      netcoreapp2.0;
+      netcoreapp2.0
     </PackageConfigurations>
     <BuildConfigurations>
-      $(BuildConfigurations);
-      $(PackageConfigurations);
+      $(PackageConfigurations)
     </BuildConfigurations>
   </PropertyGroup>
 
+  <Import Project="Directory.Build.props" />
+
+  <PropertyGroup>
+    <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)' == ''">$(TargetFramework)</NuGetTargetMoniker>
```

Comment:
Why do we need this now that we remove buildtools?

Reply:
NuGetTargetMoniker isn't unique to buildtools.  Let me look at the specific reason I needed to add this back.

Reply:
We use it in packaging:  
```
D:\j\workspace\windows-TGrou---0d2c9ac4\.packages\microsoft.dotnet.build.tasks.packaging\1.0.0-beta.19070.2\build\PackageLibs.targets(124,5): error MSB4044: The "GetPackageDestination" task was not given a value for the required parameter "NuGetTargetMoniker". [D:\j\workspace\windows-TGrou---0d2c9ac4\src\shims\shims.proj]
```

Instead I'll set PackageTargetFramework in Directory.build.props